region/lowernorfair/east/Lower Norfair Spring Ball Maze.json

```diff
@@ -479,6 +480,70 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 7],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_navigateHeatRooms",
+        {"heatFrames": 60},
+        {"or": [
+          {"and": [
+            "canDodgeWhileShooting",
+            {"enemyKill": {"enemies": [["Alcoon"]], "explicitWeapons": ["Missile", "Super", "Charge+Plasma"]}},
+            {"heatFrames": 120}
+          ]},
+          {"and": [
+            {"enemyKill": {"enemies": [["Alcoon"]], "explicitWeapons": ["Missile", "Super", "Wave", "Ice+Spazer", "Plasma", "ScrewAttack", "PseudoScrew", "PowerBomb"]}},
+            {"heatFrames": 180}
+          ]},
+          {"and": [
+            {"heatFrames": 200},
+            {"or": [
+              "canDodgeWhileShooting",
+              "Morph"
+            ]}
+          ]},
+          {"and": [
+            {"heatFrames": 160},
+            {"enemyDamage": {"enemy": "Alcoon", "type": "fireball", "hits": 1}}
+          ]}
+        ]},
+        {"or": [
+          "h_heatedCrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"or": [
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 0}]},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
+              ]},
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 3}]},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 5}}
+              ]}
+            ]}
+          ]}
+        ]},
+        {"canShineCharge": {"usedTiles": 27, "gentleUpTiles": 2, "openEnd": 0}},
+        {"or": [
+          "canDownBack",
+          "canCarefulJump"
```

Comment:
```suggestion
          "canTrickyJump"
```

Reply:
Based on https://maprando.com/logic/room/146/6/7/51 which is only careful jump? Are you thinking raise it to tricky due to carrying a shinecharge? 

Looking closer, 120 frames that the Weave strat uses leaves 60 shine frames left at [7]. Also, there are [4, 3] Leave with Shinespark strats that don't even require this.